tornado/http1connection.py

```diff
@@ -131,7 +131,8 @@ def __init__(
         self.no_keep_alive = params.no_keep_alive
         # The body limits can be altered by the delegate, so save them
         # here instead of just referencing self.params later.
-        self._max_body_size = self.params.max_body_size or self.stream.max_buffer_size
+        self._max_body_size = self.params.max_body_size if self.params.max_body_size is not None \
+                                                                else self.stream.max_buffer_size
```

Comment:
flake8 wants you to indent this line by only 4 more than the previous, like:

```python
        self._max_body_size = self.params.max_body_size if self.params.max_body_size is not None \
            else self.stream.max_buffer_size
```

but you could also wrap it differently:

```python
        self._max_body_size = self.params.max_body_size \
            if self.params.max_body_size is not None else self.stream.max_buffer_size
```

or writing it differently:

```python
        self._max_body_size = self.params.max_body_size
        if self._max_body_size is None:
            self._max_body_size = self.stream.max_buffer_size
```

Reply:
This style doesn’t look good 😔, but thank you.